prometheus_juju_exporter/config_default.yaml

```
@@ -11,3 +11,6 @@ juju:
 exporter:
   port: 9748
   collect_interval: 15
+
+machine:
```

'machine' can be a bit vague

```suggestion
detection: # parameters affecting the detection algorithm
```